.github/workflows/dependencies/run-dotnet-whatsnew.sh

```diff
@@ -7,9 +7,6 @@ set -e
 #     $2 is the <github.repository>
 #     $3 is the savedir
 
-declare -r STARTDATE=$(date "+%F" -d "$(date +'%Y-%m-01') -1 month");
-declare -r ENDDATE=$(date "+%F" -d "$STARTDATE +1 month -1 day");
-
 echo "From $STARTDATE to $ENDDATE"
```

Comment:
Will this echo statement fail since the variables are no longer defined?

Reply:
Yes. Good catch.